### ISSUE TITLE
Appveyor and Codecov

### ISSUE THREAD

**Issue by devmotion** [PieterjanRobbe/GaussianRandomFields.jl#8]
It seems Appveyor and Codecov are not enabled for this repository although configuration for both CI services exists. It would be nice to add badges for both services to README as well.

**Comment by PieterjanRobbe** [comment]
Yes, also, the README could use an update. I'll see what I can do.
